### DISCUSSION TITLE
Availability for Mac OS X

### DISCUSSION THREAD

**Discussion by Valensino** [virtualagc/virtualagc#1207] [Q&A]
I'm very excited about this project and a great fan of the lunar missions. Looking thru the wiki documents I was missing some explanations on how to build the AGC on a MacOS X, or to be more precise, on how to build the yaYUL assembler. There are some explanations how to do it on Unix/Windows, but not for Mac. Am I right or just not smart enough to find them?

**Comment by rburkey2005** [comment]
@Valensino I apologize for the late answer to a question you probably no longer care about after 3 years, but this is the first I saw your question!  Alas, I haven't found any way to get GitHub to notify me of new stuff in the discussion area.

[The download page](https://www.ibiblio.org/apollo/download.html#Mac_OS_X) gives all the info I have on the topic of Mac OS installation, divided into pre-Monterey, Monterey, and post-Monterey.
